modules/openapi-generator/src/main/java/org/openapitools/codegen/OpenAPINormalizer.java

```diff
@@ -338,6 +352,25 @@ private void normalizePaths() {
             PathItem path = pathsEntry.getValue();
             List<Operation> operations = new ArrayList<>(path.readOperations());
 
+            Map<String, Function<PathItem, Operation>> methodMap = Map.of(
+                    "get", PathItem::getGet,
+                    "put", PathItem::getPut,
+                    "head", PathItem::getHead,
+                    "post", PathItem::getPost,
+                    "delete", PathItem::getDelete,
+                    "patch", PathItem::getPatch,
+                    "options", PathItem::getOptions,
+                    "trace", PathItem::getTrace
+            );
+
+            methodMap.forEach((method, getter) -> {
```

Comment:
thanks for the PR

minor suggestion: add a line or 2 explaining what this forEach block does

Reply:
Added a comment! Thank you for the review